lib/manageiq/rpm_build/build_copr.rb

```diff
@@ -22,11 +22,14 @@ def generate_rpm
         Dir.chdir(RPM_SPEC_DIR) do
           generate_spec_from_template
 
+          rpm_dir = BUILD_DIR.join("rpms")
+          srpm_dir = rpm_dir.join("src")
```

Comment:
Would it be a problem to set srpm directory to be outside of rpm dir?  This way, you don't get srpm by default (which doubles the size) and you can separately mount the volume only when you want the srpms.

Reply:
Oh, I was planning to keep the SRPM so that we can rebuild on multiple architectures in the future.  Also for upload to the RPM repository.

Reply:
Right, I understand for our nightly/release build. I was thinking for other 'development' use cases where people most likely wants just RPMs.

Reply:
I was thinking about this... since gemset tar file includes ruby gems with compiled binaries, we can't really build RPMs for different architectures from the same source right now...

Reply:
hmm... that's a problem for other arches.  It probably doesn't make sense to publish these "srpms" since they can't be used to build anything else.